queries/gotmpl/highlights.scm

```diff
@@ -0,0 +1,125 @@
+; Identifiers
+([
+  (field)
+  (field_identifier)
+] @property
+  (#set! priority 101))
```

Comment:
What is the reason for spamming `priority` here? That is normally not needed.

Reply:
I removed them for now. @kmoschcau can you tell why you added them?

Reply:
I added these because I use background color for literals in all languages. (strings, numbers, etc.)
gotmpl building on plain text might not need this. But helm building on YAML I think needed it, so that the gotmpl specific highlights were applied on top of the highlights that YAML adds.

Reply:
Then you should _lower_ the priority of the YAML literals (see markdown).

Reply:
That definitely sounds more reasonable. :D